ui-particles-angular/projects/ui-particles-angular/src/lib/gio-license/gio-license.service.ts

```diff
@@ -89,16 +89,16 @@ export class GioLicenseService {
     return this.loadLicense$;
   }
 
-  public isMissingFeature$(licenseOptions: LicenseOptions | undefined): Observable<boolean> {
-    if (licenseOptions?.feature == null || !licenseOptions?.feature) {
+  public isMissingFeature$(feature: string | undefined): Observable<boolean> {
+    if (feature == null) {
```

Comment:
Maybe add as well if feature is undefined?

Reply:
`x == null` it's like `x === null && x === undefined`

Reply:
I'm always afraid that one day the linter will make them strict by default and change == to ===. 🙈

Reply:
Ok, so if we add the three `=`, then add a equals for undefined. If not, then it's fine as it is. 👍